interface/backend/api/views.py

```diff
@@ -195,3 +200,32 @@ def nodule_update(request, nodule_id):
     Nodule.objects.filter(pk=nodule_id).update(lung_orientation=enums.LungOrientation[lung_orientation].value)
     return Response(
         {'response': "Lung orientation of nodule {} has been changed to '{}'".format(nodule_id, lung_orientation)})
+
+
+@api_view(['POST'])
+def start_new_case(request):
+    try:
+        uri = json.loads(request.body)['uri']
+    except Exception as e:
+        return Response({'response': "An error occurred: {}".format(e)}, 500)
+
+    # drop existing case(s) with candidates and nodules
+    all_cases = Case.objects.all()
```

Comment:
Hang on, is that right? It will essentially make the entire process global whilst we would surely want it per-client?

Reply:
This a valid concern, but it's a way all our modules implemented right now. We can later add authentication and implement per-client separation.

Reply:
@lamby Bear in mind the design doc stipulates single standalone computer w/ no auth story to keep everyone focused on the value added parts of the project.